release-02/refactor/bin/control.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'pathname'
+
+if ARGV[0] == 'init'
+  if Pathname.new('minitwit.db').file?
+    puts 'Database already exists.'
+    exit
+  end
+  system('sqlite3 ./db/minitwit.db < ./db/schema.sql')
+  puts 'Putting a database to /tmp/minitwit.db...'
+  # Run app
+elsif ARGV[0] == 'start'
+  puts 'Starting MiniTwit...'
+elsif ARGV[0] == 'stop'
+  puts 'Stopping MiniTwit...'
+elsif ARGV[0] == 'inspectdb'
+
+elsif ARGV[0] == 'flag'
+
+else
+  puts 'I do not know this command...'
+end
```

Comment:
Same thing for this file, it seems that this is not even completed, so this should not be part of a PR to master. As proposed above, you should complete it (if you want) and do a new PR to master.

Reply:
File is removed.